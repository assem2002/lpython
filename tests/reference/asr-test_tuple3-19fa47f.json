{
    "basename": "asr-test_tuple3-19fa47f",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_tuple3.py",
    "infile_hash": "cfa12cc4d8456bfd45e77cc49c5e8d2417181a1443cdaaf8b15c0528",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_tuple3-19fa47f.stderr",
    "stderr_hash": "842e7a83a11ec5b255d3891c08438a4ce4b0e1a375b754d69434db93",
    "returncode": 2
}