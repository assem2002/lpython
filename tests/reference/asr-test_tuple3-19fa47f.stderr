semantic error: Tuple index out of bounds
 --> tests/errors/test_tuple3.py:3:11
  |
3 |     print(t[-4])
  |           ^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
